jujuclient/apiconnector/apiconnector.go

```diff
@@ -67,6 +68,19 @@ func (c *Connector) Connect(dialOptions ...api.DialOption) (api.Connection, erro
 	for _, f := range dialOptions {
 		f(&opts)
 	}
+
+	// By default there is no bakey client in the dial options, so we reproduce
```

Comment:
// By default there is no bakeRy client in the dial options, so we reproduce

Reply:
Done, thx